src/ctapipe/image/tests/test_hillas.py

```diff
@@ -253,6 +253,108 @@ def test_single_pixel():
     assert hillas.length.value == 0
     assert hillas.width.value == 0
     assert np.isnan(hillas.psi)
+    assert np.isnan(hillas.psi_uncertainty)
+
+
+def test_psi_uncertainty():
+    x = y = np.arange(3)
+    x, y = np.meshgrid(x, y)
+
+    geom = CameraGeometry(
+        name="testcam",
+        pix_id=np.arange(9),
+        pix_x=x.ravel() * u.cm,
+        pix_y=y.ravel() * u.cm,
+        pix_type="rectangular",
+        pix_area=np.full(9, 1.0) * u.cm**2,
+    )
+
+    # symmetric image should have large uncertainty
+    image = np.zeros((3, 3))
+    image[0, 0] = 5
+    image[0, 1] = 5
+    image[0, 2] = 5
+    image[1, 0] = 5
+    image[1, 1] = 10
+    image[1, 2] = 5
+    image[2, 0] = 5
+    image[2, 1] = 5
+    image[2, 2] = 5
+    image = image.ravel()
+    hillas = hillas_parameters(geom, image)
+    assert hillas.psi_uncertainty.value > 1e10
+
+    # check two images with identical morphology but different intensities
+    image1 = np.zeros((3, 3))
+    image1[0, 0] = 5
+    image1[1, 1] = 10
+    image1[2, 2] = 5
+    image2 = np.zeros((3, 3))
+    intensity_scaling = 2.0
+    image2 += image1 * intensity_scaling
+    image1 = image1.ravel()
+    image2 = image2.ravel()
+    hillas1 = hillas_parameters(geom, image1)
+    hillas2 = hillas_parameters(geom, image2)
+    assert (
+        abs(
+            hillas1.psi_uncertainty.value / hillas2.psi_uncertainty.value
+            - np.sqrt(intensity_scaling)
+        )
+        < 1e-4
```

Comment:
Same here, why do we just check for < 1e-4, what if it is larger? What is the unit? Why do you subtract sqrt(2)?

Why do we test on the ratio of two different hillas parameters?